src/Ormolu.hs

```diff
@@ -83,12 +85,14 @@ ormolu cfgWithIndices path str = do
     (_, result1) <-
       parseModule'
         cfg
+        fixityMap
         OrmoluOutputParsingFailed
         path
         (T.unpack txt)
     unless (cfgUnsafe cfg) $ do
       when (length result0 /= length result1) $
-        liftIO $ throwIO (OrmoluASTDiffers path [])
+        liftIO $
```

Comment:
Do you know why this is happening? `($)` is right-associative, so `liftIO $ throwIO (OrmoluASTDiffers path [])` should be grouped together and they are originally on the same line:

```haskell
      when (length result0 /= length result1) $
        liftIO $ throwIO (OrmoluASTDiffers path [])
```

however a new line is introduced after the second `$`.

Reply:
The whole `$` chain is multiline, so each element of the chain will be on its own line. That's why a line break is introduced after the 2nd `$`

Reply:
Let's keep it as it is right now.